apps/token/src/lib/url-connector.ts

```diff
@@ -0,0 +1,150 @@
+import { ethers, Wallet } from 'ethers';
+import { Connector } from '@web3-react/types';
+
+import { Eip1193Bridge } from '@ethersproject/experimental';
+import type { ConnectionInfo } from '@ethersproject/web';
+import type { Actions } from '@web3-react/types';
+import { ENV } from '../config/env';
+
+export class CustomizedBridge extends Eip1193Bridge {
```

Comment:
chore (non-blocking): I feel like this is something we'll want across apps? Perhaps it should live in libs/web3?

Reply:
I think let's leave it here for now. I imagine that there will be required changes while we get used to using this for the token frontend. I'd prefer to leave it in here until it's stabilised a bit.